src/aliases.jl

```diff
@@ -0,0 +1,22 @@
+# Various aliases to support the Base.Filesystem API or overwrite base behaviour.
+
+# Fallback non-filepath methods if folks explicitly import these symbols
+/(args...) = Base.:/(args...)
```

Comment:
Nice, so even though we are making a new `/` function we are falling back on to using `Base`s methods :+1:

Reply:
Yeah, that was the suggestion from Jeff. Apart from it not seeming like a common practice it seems like an okay solution.

https://github.com/rofinn/FilePathsBase.jl/issues/53#issuecomment-606263109